app/experimenter/nimbus-ui/src/types/globalTypes.ts

```diff
@@ -7,21 +7,30 @@
 // START Enums and Input Objects
 //==============================================================
 
-export enum NimbusChangeLogOldStatus {
+/**
+ * An enumeration.
+ */
+export enum ExperimentsNimbusChangeLogOldStatusChoices {
   COMPLETE = "COMPLETE",
   DRAFT = "DRAFT",
   LIVE = "LIVE",
   PREVIEW = "PREVIEW",
 }
 
-export enum NimbusChangeLogOldStatusNext {
+/**
+ * An enumeration.
+ */
+export enum ExperimentsNimbusChangeLogOldStatusNextChoices {
   COMPLETE = "COMPLETE",
   DRAFT = "DRAFT",
   LIVE = "LIVE",
   PREVIEW = "PREVIEW",
 }
 
-export enum NimbusDocumentationLinkTitle {
+/**
+ * An enumeration.
+ */
```

Comment:
Nit: better docs here :)

Reply:
These files are auto generated ;)

Reply:
There's no way to edit the way we generate the docs?

Reply:
Not afaik, there might be, why do you ask?